ci-operator/step-registry/cucushift/upgrade/eus/unpause-worker-mcp/cucushift-upgrade-eus-unpause-worker-mcp-commands.sh

```diff
@@ -19,32 +19,75 @@ function debug() {
     fi
 }
 
+# After upgrading, the paused mcp should not be upgraded, so its version is not same with normal mcp
+function pre_check_mcp(){
+    echo "Pre check, after upgrading, the paused mcp should not be upgraded..."
+    local paused_worker_version running_worker_version
+    paused_worker_version=$(oc get -l 'node-role.kubernetes.io/'${PAUSED_MCP_NAME}'=' nodes -ojson | jq '.items[0].status.nodeInfo' | jq -r '.containerRuntimeVersion,.kernelVersion,.osImage')
+    echo "Paused worker version: "
+    echo $paused_worker_version
+    running_worker_version=$(oc get -l 'node-role.kubernetes.io/worker=,node-role.kubernetes.io/'${PAUSED_MCP_NAME}'!=' nodes -ojson | jq '.items[0].status.nodeInfo' | jq -r '.containerRuntimeVersion,.kernelVersion,.osImage')
```

Comment:
Actually I think we do not need `pre_check_mcp` check. This step is only targeted for unpause mcp, so I suggest to keep it as clean as possible.

Reply:
Wouldn't it also checked if the normal worker has already been upgraded while the paused worker not? I think this is a key check point in canary upgrade, right?